pkg/master/master.go

```diff
@@ -177,15 +169,24 @@ func (m *Master) InstallAPIs(c *Config) {
 			Help: "The time since the last successful synchronization of the SSH tunnels for proxy requests.",
 		}, func() float64 { return float64(m.tunneler.SecondsSinceSync()) })
 	}
+	apiserver.InstallSupport(m.MuxHelper, m.RootWebService, c.EnableProfiling, healthzChecks...)
+
+	// Install v1 unless disabled.
+	if !m.ApiGroupVersionOverrides["api/v1"].Disable {
+		// Register v1 API.
+		m.initV1ResourcesStorage(c)
```

Comment:
It does not just return the set of resources. Its different because it sets a lot of fields as well.